src/lib.rs

```diff
@@ -96,22 +113,24 @@ pub struct Nitf {
 /// let nitf_path = Path::new("../example.nitf");
 /// let nitf = nitf_rs::read_nitf(nitf_path);
 /// ```
-pub fn read_nitf(path: &Path) -> Nitf {
-    let mut file = File::open(path).unwrap();
+pub fn read_nitf(path: &Path) -> NitfResult<Nitf> {
+    // Crash if failure to open file
+    let mut file = File::open(path).or(Err(NitfError::IOError))?;
```

Comment:
I feel like I might still want to `unwrap` this kind of error, since I can't 'wrap' the IO related errors with my own NITF type

Reply:
just jumping in here without knowing the specifics...

I was looking back here at the more robust example and they do wrap this exact error

https://github.com/pretzelhammer/rust-blog/blob/master/posts/tour-of-rusts-standard-library-traits.md#error-handling

Reply:
Good catch. I didn't read too deep into the `thiserror` docs, but they provide `transparent` and `#[from]` attributes for bundling other errors into a unified enum.